Functions.Templates/Templates/SqlOutputBinding-PowerShell/run.ps1

```diff
@@ -0,0 +1,28 @@
+<#
+Sample SQL Output Binding
+See https://aka.ms/sqlbindingsinput for more information about using this binding
+These tasks should be completed prior to running :
+    1. Update "commandText" in function.json - this should be the name of the table that you wish to upsert values to
+    2. Add an app setting named "SqlConnectionString" containing the connection string to use for the SQL connection
+    3. Change the bundle name in host.json to "Microsoft.Azure.Functions.ExtensionBundle.Preview" and the version to "[3.*, 4.0.0)"
+@param Request The HttpRequest that triggered this function
+#>
+using namespace System.Net
+
+# Input bindings are passed in via param block.
```

Comment:
to avoid confusion, change this comment to something like "Trigger binding data passed in via param block"
referencing input bindings in a barebones output binding sample may confuse newcomers